apps/rest/run_evadb.py

```diff
@@ -0,0 +1,86 @@
+from flask import Flask
+from flask_restful import Api, Resource
+import evadb
+
+app = Flask(__name__)
+api = Api(app)
+
+cursor = evadb.connect().cursor()
+
+class DatabaseResource(Resource):
+    def get(self,db_name):
+
+        # Create a database
+        query = """ CREATE DATABASE """ + db_name + """
+            WITH ENGINE = 'postgres',
+            PARAMETERS = {
+                "user": "eva",
+                "password": "password",
+                "host": "localhost",
+                "port": "5000",
+                "database":""" + db_name + """
+            }; """
+        
+        res = cursor.query(query).df()
+        return {"msg" : "Created Database" , "response" : res.to_json()}
```

Comment:
We do not create database ourselves. Users will use `CREATE DATABASE` to create the connection as needed.